src/hdmf/build/manager.py

```diff
@@ -474,23 +474,25 @@ def load_namespaces(self, **kwargs):
         for new_ns, ns_deps in deps.items():
             for src_ns, types in ns_deps.items():
                 for dt in types:
-                    container_cls = self.get_container_cls(src_ns, dt)
+                    container_cls = self.get_container_cls(src_ns, dt, autogen=False)
```

Comment:
I think the autogen flag is not necessary. You could just do:
```suggestion
                    container_cls = self.__get_container_cls(src_ns, dt)
```

Reply:
Never mind - this is useful since I now propose to change `__get_conainer_cls`